.circleci/config.yml

```diff
@@ -710,6 +710,12 @@ commands:
           gcloud-service-key: <<parameters.gcloud-service-key>>
           google-compute-zone: <<parameters.google-compute-zone>>
           google-project-id: <<parameters.google-project-id>>
+      - run:
+          command: |
+            gcloud components update --quiet
```

Comment:
You should probably change the version on installation rather than this separate upgrade step.

Reply:
hehe, yeah that prolly makes more sense.
I'll make the change, let CI run up the cluster deployment and merge once that passes if that's cool